ping/tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+
+@pytest.fixture(scope='session')
+def dd_environment():
+    yield
```

Comment:
```suggestion
    yield instance
```

This method is used to spin up a test environment, passing the instance parameter will add the right `config.yaml` file to the agent container.